### PR TITLE
Send alert chart labels config key to cloud

### PR DESCRIPTION
**Pull request by MrZammler**
##### Summary

This PR sends the `chart labels` key of alerts configuration to the cloud.

##### Test Plan

* Install this PR, but not start it yet. Claim it to staging. Enable the conversation log of aclk.
* Create a custom version of e.g. `disks.conf`.
* Cloud should request this new configuration (i.e. in access.log notice a `Request to send alert config`)
* In aclk.log there should be the agent's response which should contain the `chart_labels`, i.e. `Request to send alert config`

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Comment by MrZammler** [comment]
The PR will also bring in the latest `aclk-schemas` which also contains a commit of `introduce new agent command protos`. @underhood I don't know it is a problem or not (likely not).

File: database/sqlite/sqlite_db_migration.c
Comment (thiagoftsm) [inline database/sqlite/sqlite_db_migration.c:82]: Should we create an index for this new field?

Comment (thiagoftsm) [reply database/sqlite/sqlite_db_migration.c:82]: Except for this specific doubt, the rest of the code looks ok.

**Review by thiagoftsm — APPROVED** [review]
PR is working as expected, LGTM!

**Comment by ktsaou** [comment]
This PR has broken master!

There is no `cfg->set_chart_labels()`.

So this part fails to compile:

```c
    if (data->cfg.chart_labels)
        cfg->set_chart_labels(data->cfg.chart_labels);
```

We should always rebase the PR and wait for the CI to finish before we merge.

**Pull request merged**
